website/docs/relay_proxy/configure_relay_proxy.md

```diff
@@ -125,9 +125,10 @@ _Note that relay proxy is only supporting this while running inside the kubernet
 | `maxEventInMemory` | int    | `100000`                                                                                                              | If we hit that limit we will call the webhook.                                                                                                                                                                                                                                                                                                                                                          |
-| `format`           | string | `JSON`                                                                                                                | Format is the output format you want in your exported file. Available format: `JSON`, `CSV`.                                                                                                                                                                                                                                                                                                            |
+| `format`           | string | `JSON`                                                                                                                | Format is the output format you want in your exported file. Available format: `JSON`, `CSV`, `Parquet`.                                                                                                                                                                                                                                                                                                            |
 | `filename`         | string | `flag-variation-{{ .Hostname}}-{{ .Timestamp}}.{{ .Format}}`                                                          | You can use a templated config to define the name of your exported files. Available replacement are `{{ .Hostname}}`, `{{ .Timestamp}}` and `{{ .Format}`                                                                                                                                                                                                                                               |
 | `csvTemplate`      | string | `{{ .Kind}};{{ .ContextKind}};{{ .UserKey}};{{ .CreationDate}};{{ .Key}};{{ .Variation}};{{ .Value}};{{ .Default}}\n` | CsvTemplate is used if your output format is CSV.<br/>This field will be ignored if you are using another format than CSV.<br/>You can decide which fields you want in your CSV line with a go-template syntax, please check [`internal/exporter/feature_event.go`](https://github.com/thomaspoignant/go-feature-flag/blob/main/internal/exporter/feature_event.go) to see what are the fields available. |`
+| `parquetCompressionCodec` | string | `SNAPPY` | ParquetCompressionCodec is the parquet compression codec for better space efficiency. [Available options](https://github.com/apache/parquet-format/blob/master/Compression.md) |`
```

Comment:
maybe we can unify the docs here with https://gofeatureflag.org/docs/go_module/data_collection
looks similar